.github/workflows/build-with-bal-test-native.yml

```diff
@@ -17,6 +17,14 @@ on:
                 default: ''
     schedule:
         -   cron: '30 18 * * *'
+  pull_request:
```

Comment:
Seems like there is an indentation issue here. Can you fix that? (See the alignment with `schedule`)